docs/src/pages/components/autocomplete/IntegrationReactSelect.js

```diff
@@ -196,8 +249,14 @@ function Placeholder(props) {
 }
 
 Placeholder.propTypes = {
+  /**
+   * The children to be rendered.
+   */
   children: PropTypes.node,
-  innerProps: PropTypes.object,
+  /**
+   * props passed to the wrapping element for the group.
+   */
+  innerProps: PropTypes.object.isRequired,
```

Comment:
> type: The prop `innerProps` is marked as required in `Placeholder`, but its value is `undefined`.

Reply:
Guessing the upstream types are wrong then, made it optional.